### ISSUE TITLE
What? How?

### ISSUE THREAD

**Issue by vladgurovich** [zackarychapple/heimdallr#3]
I got a link to this repo from Angular Adventures podcasts, but this repo is lacking any sort of information on how is this intended to be used. Where are these metrics submitted? My google fu failed to find anything

**Comment by zackarychapple** [comment]
@vladgurovich yes, the documentation is out of date I have been meaning to update it.   For the metrics part of your question they are submitted to where-ever you desire. For instance an ELK instance where you can build dashboards around your data.
